src/main/scala/org/broadinstitute/hail/driver/AnnotateSamplesFam.scala

```diff
@@ -28,6 +28,10 @@ object AnnotateSamplesFam extends Command {
     @Args4jOption(required = false, name = "-m", aliases = Array("--missing"),
       usage = "Identifier to be treated as missing (for case-control, in addition to `0', `-9', and non-numeric)")
     var missing: String = "NA"
+
+    @Args4jOption(required = false, name = "-implicit", aliases = Array("--implicit-parents"),
```

Comment:
single-dash args should be one letter.  It's okay not to have a shortcut, and just leave name as `--implicit-parents`.